src/net/rpc.rs

```diff
@@ -125,35 +126,17 @@ impl Rpc {
 		tokio::task::spawn(async move {
 			let rpc = moved_rpc;
 			if let Some(channel) = DB.get().unwrap().notifications() {
-				while let Ok(v) = channel.recv().await {
-					trace!("Received notification: {:?}", v);
-					// Find which websocket the notification belongs to
-					match LIVE_QUERIES.read().await.get(&v.id) {
-						Some(ws_id) => {
+				while let Ok(notification) = channel.recv().await {
+					// Find which WebSocket the notification belongs to
+					if let Some(ws_id) = LIVE_QUERIES.read().await.get(&notification.id) {
+						// Check to see if the WebSocket exists
+						if let Some(websocket) = WEBSOCKETS.read().await.get(ws_id) {
+							// Serialize the message to send
+							let msg = res::success(None, notification);
+							// Get the current output format
+							let fmt = rpc.read().await.format.clone();
 							// Send the notification to the client
-							let msg_text = res::success(None, v.clone());
-							let ws_write = WEBSOCKETS.write().await;
-							match ws_write.get(ws_id) {
-								None => {
-									error!(
-										"Tracked WebSocket {:?} not found for lq: {:?}",
-										ws_id, &v.id
-									);
-								}
-								Some(ws_sender) => {
-									msg_text
-										.send(rpc.read().await.format.clone(), ws_sender.clone())
-										.await;
-									trace!(
-										"Sent notification to WebSocket {:?} for lq: {:?}",
-										ws_id,
-										&v.id
-									);
-								}
-							}
-						}
-						None => {
-							error!("Unknown websocket for live query: {:?}", v.id);
+							msg.send(fmt, websocket.clone()).await;
```

Comment:
```suggestion
							msg.send(out_fmt, websocket.clone()).await;
```
Mirrored

Reply:
I made the variable names clearer in [784f290](https://github.com/surrealdb/surrealdb/pull/2230/commits/784f29003f0581f749df5b1053cd230edc2c2eab) 😀